content/intro_video.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Introductory speech at ACM Conference"
+date = "2024-08-17"
+description = "Check out the recording at ACM Conference about Verso!"
+[taxonomies]
+tags = ["video", "conference", "ACM", "Singapore"]
++++
+
+Our main speaker [Denjell] were introducing Verso at the `ACM Conference` in Singapore on March 22, 2024.
```

Comment:
```suggestion
Our keynote speaker [Denjell] presented Verso at the ACM Conference in Singapore on 2024-03-22.
```

I would recommend to use ISO 8601 as date format. https://en.wikipedia.org/wiki/ISO_8601